### PR TITLE
update manifold

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: algorithm/cgal/install_manifold_native.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ([[ -d manifold ]] || git clone https://github.com/pentacular/manifold &&
   cd manifold &&
-  git checkout jot
+  # git checkout jot
```
Comment (BarbourSmith) [inline algorithm/cgal/install_manifold_native.sh:3]: The name change is trickling in 😀

**Review by BarbourSmith — APPROVED** [review]
I'm pretty out of the loop, but it looks good to me! 

A couple of the visual diffs look a little funky:

image

image

image

How are things going generally?

**Comment by pentacular** [comment]
> How are things going generally?

Trying to figure out why a bunch of stuff is broken after upgrading manifold. :)

It may be that I'm still not processing them quite right.

I'm also thinking of working on a cli interface for a bit to support batch processing without browser.

**Comment by pentacular** [comment]
> > How are things going generally?
> 
> Trying to figure out why a bunch of stuff is broken after upgrading manifold. :)
> 
> It may be that I'm still not processing them quite right.
> 
> I'm also thinking of working on a cli interface for a bit to support batch processing without browser.

Ok, this mostly seems to be working now. :)

**Comment by BarbourSmith** [comment]
A CLI would be pretty sweet for doing bulk jobs. That would probably open up the possibility of some new use cases

**Pull request merged**